### PR TITLE
feat(costmap_generator, control_validator, scenario_selector, surround_obstacle_checker, vehicle_cmd_gate): add processing time pub.

### PR DESCRIPTION
**Pull request by Kazunori-Nakajima**
## Description

Added processing_time_publiser for the following five
And, various processing_times can be referenced from /system/processing_time_checker/metrics.

- /planning/scenario_planning/parking/costmap_generator
- /control/control_validator
- /planning/scenario_planning/scenario_selector
- /planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker
- /control/vehicle_cmd_gate

## How was this PR tested?
Result of `ros2 topic echo /system/processing_time_checker/metrics`

/system/processing_time_checker/metrics

### Regarding `costmap_generator` taking more than 300 ms.

Sorry, we could not figure out why it is taking so long.
However, since TimeKeeper() was outputting similar results, we thought it was not a mistake in the measurement method.

Measurement resut by TimeKeeper()

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by kosuke55 — APPROVED** [review]

**Review by go-sakayori — APPROVED** [review]
LGTM for surround obstacle checker

**Comment by kosuke55** [comment]
@.reviewers
I am not sure whose approve is needed, but please check and approve

File: planning/autoware_costmap_generator/nodes/autoware_costmap_generator/costmap_generator_node.cpp [hunk 1]
```diff
@@ -277,7 +279,13 @@ void CostmapGenerator::onScenario(const tier4_planning_msgs::msg::Scenario::Cons
 void CostmapGenerator::onTimer()
 {
   autoware::universe_utils::ScopedTimeTrack scoped_time_track(__func__, *time_keeper_);
+  stop_watch.tic();
   if (!isActive()) {
+    // Publish ProcessingTime
+    tier4_debug_msgs::msg::Float64Stamped processing_time_msg;
+    processing_time_msg.stamp = get_clock()->now();
+    processing_time_msg.data = stop_watch.toc();
+    pub_processing_time_ms_->publish(processing_time_msg);
```
Comment (kosuke55) [inline planning/autoware_costmap_generator/nodes/autoware_costmap_generator/costmap_generator_node.cpp:288]: memo:
I understood that processing time must always be publised, even if the process does not run.
This is for scenario testing and graph creation.

**Review by takayuki5168 — APPROVED** [review]

**Comment by kosuke55** [comment]
@TakaHoribe @tkimura4 
could you please approve this PR?

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**